content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -1,296 +1,425 @@
+2. Run `AssetBundler.exe` from the `<engine>\build\windows_vs2019\bin\profile` directory. This opens Asset Bundler with the GUI, which we'll use in the following examples. (Alternatively, to use the CLI, run `AssetBundlerBatch.exe`.)
 
-In this example, you will create two bundles - for game assets and engine assets. The game assets contains your project's levels and all of the assets within them, such as objects, environment, materials, and so on. The engine assets contain essential files needed to load and run the Game Launcher. 
+Now you have the Asset Bundler running. It should look like this:
 
-### Build the Asset Bundler
+{{< image-width "/images/user-guide/packaging/windows-release-build/asset-bundler-default-gui.png" "1000" "An annotated image of O3DE editor's user interface." >}}
 
-1. Use CMake to build the Asset Bundler in your engine. 
+<br></br>
 
-```cmd
-cmake --build build/windows_vs2019 --target AssetBundler --config profile -- /m
-```
+{{< known-issue >}}
+There may be errors and warnings about the "AssetBundler" and "AssetSeedManager" that are listed in the console of the Asset Bundler. You can safely ignore them.
+{{< /known-issue >}}
 
 ### Create a bundle for game assets
 
-When bundling your game assets, it's only important to bundle assets that your game actually uses in its levels. There's no need to include assets in your project directory that are never loaded in your project. You can use the Asset Bundler to generate a list of assets that your levels depend on. This ensures that your resulting package file is at an optimal size. 
+When bundling your game assets, it's only important to bundle assets that your game actually uses in its levels. There's no need to include assets in your project directory that are never loaded in your project. You can use the Asset Bundler to generate a list of assets that your levels depend on. This helps ensure that your resulting package file is at an optimal size.
 
 
 #### Create a new seed asset list
-Create a new see asset list that contains only your level assets. 
 
-2. Create a new seed list file by clicking **Create a new Seed List file** in the **Seed List file** panel under the **Seeds** tab
-3. Select your new seed list file from the list. 
-4. Click **+ Add Asset** in the **Product Assets** panel. This opens the **Add Seed Asset** dialog. 
-5. In the list of platforms, enable **pc**.
-6. Click **Browse...** and navigate to the `levels` folder and choose **Add Seed**. 
-7. Verify that your seed list has the level assets in the **Product Assets** panel. 
+1. In the Asset Bundler, on the **Seeds** tab, in the **Seed List file** panel, click **Create a new Seed List file**. For this example, name the file `GameSeedList`.
```

Comment:
```suggestion
1. In the Asset Bundler GUI, on the **Seeds** tab, in the **Seed List file** panel, click **Create a new Seed List file**. For this example, name the file `GameSeedList`.
```
Because we shifted from using "the Asset Bundler" in the previous edit pass to "Asset Bundler" (proper tool name, no "the") in the latest version, I'm enforcing calling it just "Asset Bundler". However, since these procedures refer specifically to "the Asset Bundler GUI," I'm just adding "GUI" at the end of these references. If there's a better, more specific name for the GUI version of the tool, you can use that instead.